Misc/NEWS.d/next/Core and Builtins/2019-03-29-17-16-13.bpo-36473.fC_E3y.rst

```diff
@@ -0,0 +1 @@
+``dict`` iterators will now detect an iteration count overflow due to changed dict keys for ``.items()`` and ``.values()``, and raise a ``RuntimeError``.
```

Comment:
Please update `Misc/NEWS.d/next/Core and Builtins/2019-03-27-23-53-00.bpo-36452.xhK2lT.rst` instead of adding new entry.
News entry is not only for credit, but also Python users.

Reply:
Done + PR updated.